modules/gitops-release-notes-1-5-0.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-5-0_{context}"]
+= Release notes for {gitops-title} 1.5.0
+
+[role="_abstract"]
+{gitops-title} 1.5.0 is now available on {product-title} 4.8, 4.9 and 4.10.
+
+[id="new-features-1-5-0_{context}"]
+== New features
+
+The current release adds the following improvements.
+
+* This enhancement upgrades Argo CD to version *2.3.3*. link:https://issues.redhat.com/browse/GITOPS-1708[GITOPS-1708]
+
+* This enhancement upgrades Application Set to version *0.4.1*. link:https://issues.redhat.com/browse/GITOPS-1850[GITOPS-1850]
+
+* This enhancement upgrades Dex to version *2.30.3*. link:https://issues.redhat.com/browse/GITOPS-1850[GITOPS-1850]
+
+* This enhancement upgrades Helm to version *3.8.0*. link:https://issues.redhat.com/browse/GITOPS-1709[GITOPS-1709]
+
+* This enhancement upgrades Kustomize to version *4.4.1*. link:https://issues.redhat.com/browse/GITOPS-1710[GITOPS-1710]
+
+* With this update, a new channel as *latest* has been added where all the latest version releases will be updated. For GitOps v1.5.0, the operator will be pushed to *gitops-1.5*, *latest* channel, and the existing *stable* channel. From v1.6 all the latest releases will be pushed to the *latest* channel and not the *stable* channel. link:https://issues.redhat.com/browse/GITOPS-1791[GITOPS-1791]
+
+* With this update, the new CSV adds the annotation `olm.skipRange: '>=1.0.0 <1.5.0'` which means all the previous release versions will be skipped. The operator will upgrade to v1.5.0 directly. link:https://issues.redhat.com/browse/GITOPS-1787[GITOPS-1787]
+
+* With this update, the Operator updates the RH-SSO to version v7.5.1 including the following enhancements:
+
+** Allows you to login into the Argo CD using the OpenShift credentials including `kube:admin`.
+** Supports Argo CD RBAC using OpenShift groups.
+** Honors `HTTP_Proxy`, RH-SSO can be used as an SSO for Argo CD running behind a proxy. 
++
+link:https://issues.redhat.com/browse/GITOPS-1330[GITOPS-1330]
+
+* With this update, a new URL field `.host` is added to the `.status` of the Argo CD. When route or ingress is enabled (priority given to route) then the route will be displayed in the new URL field. If no URL is provided from the route or ingress, the `.host` field will not be displayed.
++
+When the route or ingress is configured, but the corresponding controller is not set up properly (i.e. is not in Ready state or does not propagate its URL), this is indicated in the Operand as well in the value of `.status.host` as `Pending` instead of the URL. This affects the overall status for the Operand by making it `Pending` instead of `Available`. link:https://issues.redhat.com/browse/GITOPS-654[GITOPS-654]
```

Comment:
```suggestion
When the route or ingress is configured, but the corresponding controller is not set up properly and is not in the `Ready` state or does not propagate its URL, the value of the `.status.host` field in the Operand indicates as `Pending` instead of the URL. This affects the overall status of the Operand by making it `Pending` instead of `Available`. link:https://issues.redhat.com/browse/GITOPS-654[GITOPS-654]
```